newshour-namedentity-wikipedialink/readme.md

```diff
@@ -1,19 +1,124 @@
+Using Wikipedia as this authoritative source is a common practice in similar technology projects, including surfacing a Wikipedia result from Google and Siri searches.  
 
-## Named Entity Linking
+### Specs
+* Annotation Project Name - `newshour-namedentity-wikipedialink`
+* Annotator Demographics   
+    * Number of annotators - 1
+    * Occupational description - College Student
+    * Age range - 20s
+    * Education - College
+* Annotation Environment Information  
+    * Name: Brat  
+    * Version: Unknown  
+    * Link/Tool Used/User Manual: (See below Tool Installation)  
+* Project Changes -  
+    * Project version: Unknown/1.0  
+    * Number of batches: 1
+    * Other version control information: None
 
-### Gold Data Generation
-The input to this annotation project was from the `newshour-namedentity` annotation project in this repository. The `golds` directory contains the public gold dataset, generated using `process.py`. Files are in .tsv format. The relevant data fields are summarized below:
+## Tool Installation: Brat
+brat rapid annotation tool    
+[Brat and Installation Instructions](https://brat.nlplab.org/index.html)
 
-- __guid__: The AAPB GUID of the annotated transcript. (_string_)
-- __anno_id__: The text-bound annotation ID. (_string_)
-- __type__: The entity category-- person, location, event, organization, title (_string_)
-- __begin_offset__: The character offset beginning the text span. (_int_)
-- __end_offset__: The character offset ending the text span. (_int_)
-- __text__: The entity. (_string_)
-- __type__: The entity category. (_string_)
-- __wiki_url__: The Wikipedia URL grounding the entity. (_string_)
-- __qid__: The Wikidata URI linking the entity via Q identifier. (_string_)
+## Annotation Guidelines: Transcribing/Closed Captioning
+> [!Important]  
+> This project builds upon Named Entity Project [aka the previous project](https://github.com/clamsproject/aapb-annotations/blob/main/newshour-namedentity/guidelines.md).  
+> The `guidelines.md` for [this project](https://github.com/clamsproject/aapb-annotations/blob/main/newshour-namedentity-wikipedialink/guidelines.md) is currently an exact copy of the one previous. 
+### Preparation
+Import the transcripts through brat. This tool seems to require a mouse.  
+`.txt` transcript that has already had named entities annotated in brat.   
+`.tab` file for collecting the annotations with duplicate entities squashed into a number. 
 
-note: for purposes of evaluation, the subtypes of the 'title' category are collapsed into one group.
\ No newline at end of file
+#### Preprocessing before Raw Annotation
+_TODO: Locate the preprocessing files for linking._  
+The input to this annotation project was from the `newshour-namedentity` project in this repository. 
+It is assumed either the brat tool handles the deduplication or a pre-processing for raw annotation in this project does that. 
```

Comment:
The tool is in fact not brat again, it is a special tool up above. 
The preprocessing might be in there.